docs/source/guide/zne-3-options.myst

```diff
@@ -212,6 +216,88 @@ folding method. Fidelity values must be between zero and one.
 
 +++
 
+### Identity Scaling
+
+Mitiq has only one noise scaling function for the *identity scaling* technique. The goal of this technique is to insert
+layers of identity gates such that they act evenly across all qubits in the circuit. A unitary $G$ is mapped as follows after
+a single identity layer is inserted :
+
+$$G \longrightarrow G I.$$
+
+Here, $G$ is one layer of gates in the input circuit where the gates in each layer are supposed to act at the same time instance.
+
+The Mitiq function for *identity insertion scaling* is {func}`.insert_id_layers()`. Depending on the `scale_factor`, the number of
+identity layers are inserted in such a way that the new scale factor is exactly the desired `scale_factor` or is very close to it. 
```

Comment:
```suggestion
To use this method, call {func}`.zne.scaling.insert_id_layers()` with both a circuit to scale, and a scale factor.
Alternatively, pass it as the `scale_noise` argument to {func}`.zne.execute_with_zne` to use it instead of the default method of {func}`.fold_gates_at_random`.
```